docs/development_suite/deploy/configure-github-actions.md

```diff
@@ -0,0 +1,126 @@
+---
+id: configure-github-actions
+title:  Configure deployment with GitHub
+sidebar_label: Configure deployment with GitHub
+slug: "/development_suite/deploy/"
+---
+The Mia-Platform Console can be configured to deploy your project through GitHub.
+
+In order to enable your project to be deployed using GitHub, you should edit the configuration of the project through the Console CMS 
+(if you do not have access to it, ask your instance administrator). In particular, the `environments.deploy` object 
+of the configuration should contain the following information:
+
+```json
+{
+  "deploy": {
+    "type": "github-actions",
+    "providerId": "your-github-provider-id"
+  }
+} 
+```
+
+:::info
+if you haven't set up a GitHub provider yet, [see this page](/development_suite/set-up-infrastructure/configure-provider) to learn how to configure it.
+Remember that the Access Token configured to your provider needs to have the `admin` scope of the organization.
```

Comment:
Why the admin scope? It is not possible to set less scope?

Reply:
from the documentation 
> All actions against organization webhooks require the authenticated user to be an admin of the organization being managed. Additionally, OAuth tokens require the admin:org_hook scope.

[Link](https://docs.github.com/en/rest/orgs/webhooks?apiVersion=2022-11-28)

Reply:
We need to use organization wide webhooks or project level webhooks?

Reply:
under the deployment_status event i found this
> To subscribe to this event, a GitHub App must have at least read-level access for the "Deployments" repository permission